tests/test_run_worker.py

```diff
@@ -1,30 +1,32 @@
 # @@@SNIPSTART hello-world-project-template-python-tests
-import asyncio
 import uuid
 
 import pytest
-from run_worker import SayHello, say_hello
+
+from activities import say_hello
+from run_worker import SayHello
 from temporalio import activity
-from temporalio.client import Client
 from temporalio.worker import Worker
+from temporalio.testing import WorkflowEnvironment
 
 
 @pytest.mark.asyncio
-async def test_execute_workflow(client: Client):
+async def test_execute_workflow():
     task_queue_name = str(uuid.uuid4())
+    async with await WorkflowEnvironment.start_local() as env:
```

Comment:
```suggestion
    async with await WorkflowEnvironment.start_time_skipping() as env:
```

And on the next test too. Our full local Temporal instance does take a bit to shutdown sometimes, and it's nice to show time skipping as the recommended test server approach anyways.